### PR TITLE
add context and remove youtube flow from home 

### LINKED ISSUE #32: Add React Context

### PR DESCRIPTION
**Pull request by lilywuyanru**
Closes: #32

Clean up code base and add context
remove youtube flow from home page

### CODE REVIEW

File: src/Room/Room.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const Room = ({ roomName, room, handleLogout }) => {
   const [participants, setParticipants] = useState([]);
   const [vid, setVid] = useState(false);
   const [mic, setMic] = useState(false);
-  const isYoutube = roomName.includes("-YT-") ? true : false;
+  const [isYoutube, setIsYoutube] = useState(false);
```
Comment (lilywuyanru) [inline src/Room/Room.js:33]: if you change this to true, you can see the Youtube room layout

Comment (matthewwong525) [reply src/Room/Room.js:33]: does the youtube work for you?

Comment (matthewwong525) [reply src/Room/Room.js:33]: actually, its an issue with the place we're pulling Youtube Metadata from in Search.js

**Review by matthewwong525 — APPROVED** [review]
Looks good, fixing Search.js will be another issue regarding the problem with Youtube

**Pull request merged**